client/extensions/woocommerce/state/sites/shipping-zone-locations/reducer.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_SHIPPING_ZONE_LOCATIONS_REQUEST,
+	WOOCOMMERCE_SHIPPING_ZONE_LOCATIONS_REQUEST_SUCCESS,
+} from 'woocommerce/state/action-types';
+import { LOADING } from 'woocommerce/state/constants';
+import { createReducer } from 'state/utils';
+
+export default createReducer( {}, {
+	[ WOOCOMMERCE_SHIPPING_ZONE_LOCATIONS_REQUEST ]: ( state, { zoneId } ) => {
+		return { ...state,
+			[ zoneId ]: LOADING,
```

Comment:
I like to keep `[ key ]: value` declarations in their own line. Too many weird symbols together if I keep them in the same line, too `Perl`-y for my taste :)